src/Storage/Commands.Storage/File/Cmdlet/SetAzureStorageFileContent.cs

```diff
@@ -80,6 +80,12 @@ public override void ExecuteCmdlet()
                 throw new FileNotFoundException(string.Format(CultureInfo.CurrentCulture, Resources.SourceFileNotFound, this.Source));
             }
 
+            // if FIPS policy is enabled, must use native MD5
+            if (FIPSEnabled)
+            {
+                CloudStorageAccount.UseV1MD5 = false;
```

Comment:
Is this the only cmdlet that would be impacted?  Wouldn't every blob-related cmdlet need this setting?

Reply:
Only upload blob/file cmdlets need use MD5 function to calculate Property ContentMD5. Other cmdlets don't use this function. So only need change cmdlets Set-AzureStorageBlob(File)Content